components/movementsensor/adxl345/adxl345.go

```diff
@@ -377,13 +377,14 @@ func getInterruptConfigurations(cfg *Config) map[byte]byte {
 			// Clear the single tap bit in the map to send the signal to pin INT1.
 			intMap &^= interruptBitPosition[singleTap]
 		}
+
 	}
 
 	return map[byte]byte{intEnableAddr: intEnabled, intMapAddr: intMap}
 }
 
 // This returns a map from register addresses to data which should be written to that register to configure single tap.
-func getSingleTapRegisterValues(singleTapConfigs *TapConfig) map[byte]byte {
+func getSingleTapRegisterValues(singleTapConfigs *TapConfig, logger logging.Logger) map[byte]byte {
```

Comment:
I believe the test files will need to change, run `make lint-go` and `make test-go` in a canon shell to make sure you're building the test files with the new function signatures.

Reply:
The tests don't call this so we are good, fixed the lint issue